src/raster/r.maxent.train/r.maxent.py

```diff
@@ -0,0 +1,1064 @@
+
+# %option
+# % key: hingethreshold
+# % type: integer
+# % label: Theshold for hinge feature
+# % description: Number of samples at which hinge features start being used.
+# % answer: 15
+# % guisection: Advanced
+# %end
+
+# %option
+# % key: beta_threshold
+# % type: double
+# % label: Regularization parameter for treshold features
+# % description: Regularization parameter to be applied to all threshold features; negative value enables automatic setting.
+# % answer: -1.0
+# % guisection: Advanced
+# %end
+
+# %option
+# % key: beta_categorical
+# % type: double
+# % label: Regularization parameter for categorical features
+# % description: Regularization parameter to be applied to all categorical features; negative value enables automatic setting.
+# % answer: -1.0
+# % guisection: Advanced
+# %end
+
+# %option
+# % key: beta_lqp
+# % type: double
+# % label: Regularization parameter for lin, quad en prod features
+# % description: Regularization parameter to be applied to all linear, quadratic and product features; negative value enables automatic setting.
+# % answer: -1.0
+# % guisection: Advanced
+# %end
+
+# %option
+# % key: beta_hinge
+# % type: double
+# % label: Regularization parameter for hinge features
+# % description: Regularization parameter to be applied to all linear, quadratic and product features; negative value enables automatic setting.
+# % answer: -1.0
+# % guisection: Advanced
+# %end
+
+# %option
+# % key: defaultprevalence
+# % type: double
+# % label: Default prevalence of the species
+# % description: Default prevalence of the species: probability of presence at ordinary occurrence points. See Elith et al., Diversity and Distributions, 2011 for details.
+# % answer: 0.5
+# % options: 0-1
+# % guisection: Advanced
+# %end
+
+# %option G_OPT_M_NPROCS
+# % key: threads
+# % label: Number of processor threads to use.
+# %end
+
+# %option G_OPT_F_BIN_INPUT
+# % key: maxent
+# % label: Location Maxent jar file
+# % description: Give the path to the Maxent executable file (maxent.jar)
+# % required: no
+# %end
+
+# %option G_OPT_MEMORYMB
+# % Description: Maximum memory to be used by Maxent (in MB)
+# %end
+
+# %flag
+# % key: r
+# % label: skipifexists
+# % description: If output files already exist for a species being modeled, skip the species without remaking the model.
+# %end
+
+# %flag
+# % key: m
+# % label: Show Maxent GUI
+# % description: Show the Maxent interface and don't run automatically.
+# %end
+
+# %flag
+# % key: s
+# % label: randomseed
+# % description: If selected, a different random seed will be used for each run, so a different random test/train partition will be made and a different random subset of the background will be used, if applicable.
+# %end
+
+# %rules
+# % exclusive: replicates,randomtestpoints
+# %end
+
+# %rules
+# % requires_all: fadebyclamping, doclamp
+# %end
+
+# import libraries
+import os
+import sys
+import subprocess
+import grass.script as gs
+import csv
+import numpy as np
```

Comment:
```suggestion
import csv
import os
import subprocess
import sys

import numpy as np

import grass.script as gs
```
My understaning is that there is a recommended order for imports:
1) builtin modules starting with "import"
2) "from" imports of elements of builtin modules
3) same as above for non-standard libraries (external dependencies)
4) same for "project" imports...
I guess alphabetical order is recommended. Others know the guidelines in more detail. They were not very strictly followed in the past allways, but it is never to late...